src/Panel/Request.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Kirby\Panel;
+
+use Kirby\Http\Response;
+
+/**
+ * @package   Kirby Panel
+ * @author    Bastian Allgeier <bastian@getkirby.com>
+ * @link      https://getkirby.com
+ * @copyright Bastian Allgeier
+ * @license   https://getkirby.com/license
+ */
+class Request
+{
+	/**
+	 * Renders request responses
+	 */
+	public static function response($data, array $options = []): Response
+	{
+		$data = Json::responseData($data);
+
+		return Panel::json(Json::responseData($data), $data['code'] ?? 200);
```

Comment:
Is it intended that `$data` is passed twice through `Json::responseData()`? If so, can we add a comment to explain why?

Reply:
It's just sloppiness after I refactored the code.